Grid/grid.css

```diff
@@ -0,0 +1,33 @@
+.grid-container{
+    width: 90%;
+    display: grid;
+    padding: 2em;
```

Comment:
I think following a single unit is the best way to go, either you follow rem or em. Let me know your thoughts on this

Reply:
yeah i hv used rem only for more than 10px units it was typo. thanks for pointing out i hv corrected that